database/src/lib.rs

```diff
@@ -71,20 +71,64 @@ impl<T> SqlxResultExt<T> for Result<T, sqlx::Error> {
     }
 }
 
-#[cfg(test)]
-mod test_util {
+pub mod test_util {
```

Comment:
Why did you remove `#[cfg(test)]` here?

Reply:
Because crates aren't recompiled with test mode on in the same workspace. These utilities need to be available to cosmicverge-server. With `#[cfg(test)]` and `pub mod test_util` the symbol still isn't available to other crates.

Reply:
Hm, hide it behind a feature called `test` and then add in `dev-dependencies` `database = { path = "../database", features = ["test"] }`.
I believe if we built some more extensive end-to-end tests in the future we are gonna need that anyway.

Reply:
These functions need to be available to other crates, and `#[cfg(test)]` prevents the module from being exported.